mapper/dm_agent/src/batcher.rs

```diff
@@ -83,19 +57,25 @@ pub struct MessageBatcher {
     sender: UnboundedSender<MeasurementGrouper>,
     mqtt_client: Arc<dyn MqttClient>,
     topic_filter: TopicFilter,
```

Comment:
you could also rename this to `source_topic_filter`. makes it easier to create the struct in `new`.